packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -158,20 +167,33 @@ contract Reserve is
 
   /**
    * @notice Set the ratio of reserve that is spendable per day.
-   * @param ratio Spending ratio as unwrapped Fraction.
+   * @param spendingRatios Spending ratio as unwrapped Fraction.
    */
-  function setDailySpendingRatio(uint256 ratio) public onlyOwner {
-    spendingRatio = FixidityLib.wrap(ratio);
-    require(spendingRatio.lte(FixidityLib.fixed1()), "spending ratio cannot be larger than 1");
-    emit DailySpendingRatioSet(ratio);
+  function setDailySpendingRatio(address[] memory tokenAddresses, uint256[] memory spendingRatios)
```

Comment:
I don't think we need to pass an array of addresses and tokens here, calling the function for each one should be fine.